subprojects/core/src/test/groovy/org/gradle/api/internal/changedetection/changes/DefaultTaskArtifactStateRepositoryTest.groovy

```diff
@@ -120,7 +120,7 @@ class DefaultTaskArtifactStateRepositoryTest extends AbstractProjectBuilderSpec
 
     def "artifacts are not up to date when any output file no longer exists"() {
         given:
-        execute(task)
+        executeTasks(task)
```

Comment:
Why not keep this `execute()`?

Reply:
It was clashing with the super class method `execute` which didn't take a var arg argument. I now solved the problem by overwriting the super class `execute` method.